easydriver/easydriver.py

```diff
@@ -249,7 +249,7 @@ def _step_once(self):
         steps {int} -- number of steps
         direction {StepDirection} -- direction to step in
     """
-    def step(self, steps, direction=StepDirection.FORWARD):
+    def step(self, steps=1, direction=StepDirection.FORWARD):
```

Comment:
Is there any particular reason we're leaving the step direction as an optional parameter?

Reply:
Not really, it was kind of an arbitrary decision.